cscareerhub/megabot
# [misc] Update dependencies and configs, restructure packages, update scripts

## Purpose of pull request:

Update dependencies and configs, restructure packages, update scripts

## Pull request checklist:

[x] Branch is named correctly
[x] Updated relevant documentation
[ ] Tested with a tester bot

## Testing instructions:

1. Run `yarn build`
2. Each package should output a `lib` folder based on `src`
3. Running `yarn start` and `yarn serve` won't work yet, but the build part should
4. `yarn test` and `yarn coverage` should try to execute jest on each package, but won't work because there are no tests yet.
